spec/factories/administration.rb

```diff
@@ -74,8 +74,8 @@
   end
 
   factory :map_location do
-    latitude { 51.48 }
-    longitude { 0.0 }
+    latitude { 40.416775 }
+    longitude { -3.703790 }
```

Comment:
Maybe we could create investments on map locations based on their headings so we don't have to rely on duplicate coordinates :thinking:, and we can keep the more-or-less-neutral Greenwich location for general map locations. Here's a possible patch:

```diff
diff --git a/spec/factories/administration.rb b/spec/factories/administration.rb
index 1203aa82c9..415e2c148b 100644
--- a/spec/factories/administration.rb
+++ b/spec/factories/administration.rb
@@ -74,8 +74,8 @@ FactoryBot.define do
   end
 
   factory :map_location do
-    latitude { 40.416775 }
-    longitude { -3.703790 }
+    latitude { 51.48 }
+    longitude { 0.0 }
     zoom { 10 }
 
     trait :proposal_map_location do
diff --git a/spec/factories/budgets.rb b/spec/factories/budgets.rb
index b1e4cd2752..f3f27a192a 100644
--- a/spec/factories/budgets.rb
+++ b/spec/factories/budgets.rb
@@ -198,7 +198,11 @@ FactoryBot.define do
     end
 
     trait :with_map_location do
-      map_location
+      map_location do
+        association :map_location,
+                    longitude: heading.longitude.to_f + rand(-0.001..0.001),
+                    latitude: heading.latitude.to_f + rand(-0.001..0.001)
+      end
     end
 
     trait :flagged do
```

The `rand` part is there so we keep creating investments on slightly different places, as we did before these changes.